### ISSUE TITLE
Settings dialog comes up in the current tab

### ISSUE THREAD

**Issue by Barry64** [sneakypete81/updatescanner#168]
* Firefox version:  57.0.1
 * Update Scanner version: 4.1.0

While Update Scanner is visible in the sidebar, if you right-click on one of the pages in the scan list and select "Settings", the resulting dialog will replace whatever is in the currently focused tab, without the user necessarily realizing it.  Better behavior would be to have the dialog be in a modal box, or to at least have it appear in a new tab.

**Comment by sneakypete81** [comment]
Good point. A modal dialog isn't possible, but opening in a new tab would be better behavior.

**Comment by gl1000007** [comment]
+1
